modules/contributing-quick-starts.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * web_console/creating-quick-start-tutorials.adoc
+
+[id="contributing-quick-starts_{context}"]
+= Contributing quick starts
```

Comment:
Hi, @ahardin-rh. Instead of documenting adding quick starts to the console-operator repo, we should document creating a `QuickStart` custom resource. This allows admins to add quick starts to their own clusters without needing to add them as default quick starts to the product.